kithara/model/hf_compatibility/to_huggingface.py

```diff
@@ -136,9 +144,36 @@ def save_lora_files(
     )
 
 
+def create_huggingface_hub_repo_if_not_exist(repo_id, repo_type):
+    if not repo_exists(repo_id, repo_type=repo_type):
+        api = HfApi()
+        api.create_repo(
+            repo_id=repo_id,
+            repo_type=repo_type,
+            exist_ok=True,
+            private=True,
+        )
+        print(f"\n Created new HuggingFace Hub {repo_type} repo: {repo_id}.")
+
+
+def copy_local_file_to_huggingface_hub(local_path, file_name, repo_id, repo_type):
```

Comment:
Better use "upload" than "copy" to be consistent to the term both the hf API uses and the GCS interface uses.